rq_dashboard/web.py

```diff
@@ -419,6 +420,14 @@ def serialize_current_job(job):
     )
 
 
+def get_worker_current_job(worker):
+    try:
+        job = worker.get_current_job()
+    except NoSuchJobError:
+        job = None
```

Comment:
I don't like swallowing this error. At the very least, let's log a warning

please see [this slack convo](https://hostcompliance.slack.com/archives/C5PU4KT38/p1564077358051600)

Reply:
@lucasnad27 I had added a warning.